### DISCUSSION TITLE
Tidy up `unprocessedRequestWarning` in popup panel

### DISCUSSION THREAD

**Discussion by partingscientist** [uBlockOrigin/uBlock-issues#2696] [Ideas]
<details>

<summary>I found this warning recently in the popup panel.</summary>

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/115052854/45b125c2-7782-4fe2-8572-7b7be435f7ad)

</details>

Obviously I don't mean to nitpick, but perhaps tidying up this warning might not be a bad idea if there's  spare time to do so. Left, right, and top margins at the very least perhaps?

**Comment by ghost** [comment]
![image](https://github.com/uBlockOrigin/uBlock-issues/assets/886325/c9b10310-adb4-453c-888e-3ff0176534c8)

Rendering on your side seems broken.

**Reply by partingscientist** [reply]
I was updating from 1.41.2 to the latest stable to get that warning. I don't know, maybe it's broken due to such a big version jump perhaps. Must be on my end only.
